src/dotnet-reqube/Models/Options.cs

```diff
@@ -16,5 +16,10 @@ internal class Options
         [Option('d', "directory", Required = false, HelpText = "Directory where reports will be saved. Working directory will be used if not set.")]
         // ReSharper disable once UnusedAutoPropertyAccessor.Global
         public string Directory { get; set; }
+
+        [Option('p', "project", Required = true,
```

Comment:
I believe that the parameter should be optional?

Reply:
Of course, I just fixed it. I had some issues with the parameters in combination with dotnet run's parameters, so I played around a bit and finally kept it in the wrong setting. I pushed again to my branch with the fix.